docs/guide/writing-tasks.md

```diff
@@ -501,7 +501,7 @@ publish_task:
   script: yarn run publish
 ```
 
-!!! tip "Task Names"
+??? tip "Task Names and Aliases"
     It is possible to specify task name via `name` field. `lint_task` syntax is simply a syntactic sugar that will be
```

Comment:
```suggestion
    It is possible to specify the task's name via the `name` field. `TASKNAME_task` syntax is simply a syntactic sugar that will be
```

Reply:
the `lint_task` pairs with the code example below to show how it expands.

Reply:
Got it.